tools/tests/unit/setupJest.js

```diff
@@ -1,2 +1,5 @@
+// TODO: see if we can make this import conditional based on node version
```

Comment:
Nuke your comment here, I don't think that's what you mean.

Reply:
sure thing